intents_consolidator/src/intents/consolidator/handlers/facial_recognition_handler.cpp

```diff
@@ -15,16 +16,32 @@ namespace consolidator
 {
 
 FacialRecognitionResult::FacialRecognitionResult(
-  const machine_intents_interfaces::msg::FacialRecognitionResult * msg)
+  const machine_intents_interfaces::msg::FacialRecognitionResult * msg, const entity::Pose pose)
 {
   embedding_ = std::make_unique<datastructures::Embedding>(msg->embedding);
+  pose_ = pose
```

Comment:
Use the initializer list:
```
FacialRecognitionResult::FacialRecognitionResult : pose_{pose} ...
```

Reply:
Can you clarify why we would use an initializer list here?

Reply:
Basically, whenever you have a member variable that you want to initialize inside a constructor, you should use the initializer list. Review this lesson for more: https://www.learncpp.com/cpp-tutorial/constructor-member-initializer-lists/